app/package.json

```diff
@@ -42,22 +50,34 @@
     "store2": "^2.13.2",
     "styled-components": "^5.3.5",
     "validator": "^13.7.0",
-    "xml2js": "^0.4.23"
+    "xml2js": "^0.4.23",
+    "yup": "^1.3.2"
   },
   "devDependencies": {
+    "@octokit/types": "^12.1.1",
+    "@types/deep-diff": "^1.0.5",
     "@types/easy-soap-request": "^4.1.1",
     "@types/jsonwebtoken": "^8.5.8",
     "@types/jwk-to-pem": "^2.0.1",
     "@types/jws": "^3.2.4",
     "@types/lodash.get": "^4.4.7",
+    "@types/lodash.isstring": "^4.0.8",
+    "@types/lodash.kebabcase": "^4.1.9",
     "@types/lodash.map": "^4.6.13",
+    "@types/lodash.omit": "^4.5.9",
+    "@types/lodash.pick": "^4.4.9",
+    "@types/mustache": "^4.2.4",
     "@types/pg": "^8.6.5",
     "@types/pg-format": "^1.0.2",
     "@types/react": "^18.0.10",
     "@types/validator": "^13.7.4",
     "@types/xml2js": "^0.4.11",
+    "@types/yup": "^0.32.0",
     "eslint": "^8.16.0",
     "eslint-config-next": "12.1.6",
+    "prettier": "^3.0.3",
+    "prettier-plugin-prisma": "^5.0.0",
+    "prisma": "^5.5.2",
```

Comment:
Should prisma be a regular dependency? Or does it only need @prisma/client

Reply:
prisma is just a cli